app/models/manageiq/providers/openstack/cloud_manager/vm/resize.rb

```diff
@@ -1,9 +1,13 @@
 module ManageIQ::Providers::Openstack::CloudManager::Vm::Resize
-  def validate_resize
-    msg = validate_vm_control
-    return {:available => msg[0], :message => msg[1]} unless msg.nil?
-    return {:available => true, :message => nil} if %w(ACTIVE SHUTOFF).include?(raw_power_state)
-    {:available => false, :message => _("The Instance cannot be resized, current state has to be active or shutoff.")}
+  extend ActiveSupport::Concern
+
+  included do
+    supports :resize do
+      unsupported_reason_add(:resize, unsupported_reason(:control)) unless supports_control?
+      unless %w(ACTIVE SHUTOFF).include?(raw_power_state)
```

Comment:
`supports_resize?` is never true now, because `%w(ACTIVE SHUTOFF)` is `["ACTIVE", "SHUTOFF"]` and not `[ACTIVE, SHUTOFF]` which would (actually still not) evaluate to the right `%w(on off)` array which corresponds to the actual values :).

But `[ACTIVE, SHUTOFF]` doesn't work within `supports...`, it won't find the constant.

So... I suggest fixing this to just say `%w(on off)`.

@gauravaradhye , @durandom Can you take care of this please? :)

Reply:
@himdel  - sorry, neither @gauravaradhye nor me are working on the codebase anymore. I'm sure one of the openstack provider maintainers can tackle that. I'm still avail for questions :)

Reply:
Ah, thanks for the heads up :) don't worry about it, the issue is there, and if it stays there I'll just create a PR one of those days ;). Good luck :)